packages/suite-desktop/src/support/DesktopUpdater/Downloading.tsx

```diff
@@ -43,11 +43,17 @@ const Text = styled(H2)`
 
 interface Props {
     hideWindow: () => void;
-    progress?: UpdateProgress;
+    progress?: Partial<UpdateProgress>;
 }
 
+const ellipsisArray = new Array(3).fill('.');
+
 const Downloading = ({ hideWindow, progress }: Props) => {
     const cancelUpdate = useCallback(() => window.desktopApi!.cancelUpdate(), []);
+    const [step, setStep] = useState(0);
+    useEffect(() => {
+        setTimeout(() => setStep(step > 2 ? 0 : step + 1), 300);
+    }, [step]);
```

Comment:
to prevent state update on unmounted component:
```suggestion
    useEffect(() => {
        const timer = setTimeout(() => setStep(step > 2 ? 0 : step + 1), 300);
        return () => clearTimeout(timer);
    }, [step]);
```

Reply:
8c7f26b